### PR TITLE
Add a docs contributing guide to the README

### PR DESCRIPTION
**Pull request by settermjd**
|    Q          |   A
|-------------- | ------
| Documentation | **yes**
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

My intention in doing this is to help others get started easier than I did the first time (this time) by including a clear set of instructions that step them through how to build the documentation locally, so that they know that they changes which they contribute will render properly, as they expect them to. 

A **big** thanks to @froschdesign for helping me get started.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +20,76 @@ $ composer require laminas/laminas-form
...
+
+The documentation for the repository is available in the "**docs/**" directory.
+It is written in [Markdown format] and built using [MkDocs].
+
+#### Testing Your Changes Locally
+
+We strongly encourage you to test your changes locally before contributing them.
+However, to do so, you need to do three things; these are:
+
+1. Have MkDocs installed
```
Comment (samsonasik) [inline README.md:33]: need `.` for consistency in last word with number 2 and 3.

File: README.md [hunk 2]
```diff
@@ -20,8 +20,76 @@ $ composer require laminas/laminas-form
...
+2. Have the Laminas documentation theme available to the laminas-form directory.
+3. Update `mkdocs.yml` to build the docs using the documentation theme.
+
+To complete these steps, please follow the instructions below.
+
+1. [Install MkDocs]
```
Comment (samsonasik) [inline README.md:39]: need `.` for consistency in last word with number 2 and 3.

**Comment by froschdesign** [comment]
@weierophinney 
This is a very good extension and we need the explanation. (The path with Docker should be added too.)
But I would move this the separate repository like `documenation`. There we can add some more content like a documentation writer's guide.

**Comment by weierophinney** [comment]
> @weierophinney
> This is a very good extension and we need the explanation. (The path with Docker should be added too.)
> But I would move this to a separate repository like `documentation`. There we can add some more content like a documentation writer's guide.
> All contents can be displayed under docs.laminas.dev and linked in the readme files.

I would argue for:

- Put this information in the `documentation` repository.
- Add a section in the [laminas/.github](https://github.com/laminas.github) `CONTRIBUTING.md` file titled "Contributing Documentation" that summarizes the "we use Markdown and MkDocs" information, and then links to the larger write-up in the `documentation` repository. (And do this in each of the `.github` repos of each organization)

**Comment by froschdesign** [comment]
> * Add a section in the [laminas/.github](https://github.com/laminas.github) `CONTRIBUTING.md`

Very good, I hadn't thought of that at all. 👍

**Comment by settermjd** [comment]
Thanks for the feedback, everyone. I'll get it done.

**Comment by settermjd** [comment]
> @weierophinney
> This is a very good extension and we need the explanation. (The path with Docker should be added too.)
> But I would move this to a separate repository like `documentation`. There we can add some more content like a documentation writer's guide.
> All contents can be displayed under docs.laminas.dev and linked in the readme files.

Did you mean the "documentation-theme" repository? Otherwise, I'm not sure which repository you mean.

**Comment by weierophinney** [comment]
> Did you mean the "_documentation-theme_" repository? Otherwise, I'm not sure which repository you mean.

We had a [documentation repo in the ZF org](https://github.com/zendframework/documentation) for tracking projects. I never migrated it here, as it didn't have code, mostly projects...

@froschdesign I can _transfer_ that repo to the Laminas org, if you'd like; that would give us the projects. I hadn't thought of that, as everything else we were migrating, and then archiving. Let me know if that's what you'd like, or if you'd like me to create a documentation repo.

**Comment by settermjd** [comment]
> > Did you mean the "_documentation-theme_" repository? Otherwise, I'm not sure which repository you mean.
> 
> We had a [documentation repo in the ZF org](https://github.com/zendframework/documentation) for tracking projects. I never migrated it here, as it didn't have code, mostly projects...
> 
> @froschdesign I can _transfer_ that repo to the Laminas org, if you'd like; that would give us the projects. I hadn't thought of that, as everything else we were migrating, and then archiving. Let me know if that's what you'd like, or if you'd like me to create a documentation repo.

I'm all for it being migrated, @weierophinney

**Comment by froschdesign** [comment]
@weierophinney 
> I can _transfer_ that repo to the Laminas org, if you'd like…

That would be great. Thanks in advance!

**Comment by weierophinney** [comment]
> That would be great. Thanks in advance!

DONE! https://github.com/laminas/documentation

I was wrong - no projects, just issues. But they're all there now.

**Comment by settermjd** [comment]
> > That would be great. Thanks in advance!
> 
> DONE! https://github.com/laminas/documentation
> 
> I was wrong - no projects, just issues. But they're all there now.

Thanks for doing that, @weierophinney.

**Comment by settermjd** [comment]
I moved the contributing documentation to the documentation repository in https://github.com/laminas/documentation/pull/15.

**Comment by froschdesign** [comment]
@settermjd 
> I moved the contributing documentation to the documentation repository in [laminas/documentation#15](https://github.com/laminas/documentation/pull/15).

This is the right place for it. The link is already included in the [_global_ contributing guidelines](https://github.com/laminas/.github/blob/master/CONTRIBUTING.md) for this organisation.

The link to the guidelines is added under each comment and issue text field and in the [community section](https://github.com/laminas/laminas-form/community) of the repository.

To avoid duplication, we should use one place for the text and close this pull request. Improvements for text should be added to the [documentation repository](https://github.com/laminas/documentation).

We already use the content so nothing is lost.

Thanks!

**Pull request closed without merging**